Makefile

```diff
@@ -155,11 +158,15 @@ lint: lint-gosec
 
 .PHONY: install-glide
 install-glide:
-	@which glide > /dev/null || go get github.com/Masterminds/glide
+	@echo "Jaeger has migrated to dep, please run 'make install-dep'" 1>&2
```

Comment:
this target should be removed

Reply:
he said to fall back to dep, which this doesn't. Also, this looks like it's always been an internal target, not something used outside of make, so I don't see much value in keeping just the name.